_organizations/eyeofbeholder-nlesc.md

```diff
@@ -0,0 +1,7 @@
+---
+title: The Eye of the Beholder Project
+homepage: https://github.com/EyeofBeholder-NLeSC
+avatar: https://avatars.githubusercontent.com/u/99891299?v=4
+---
+The Netherlands eScience Center
```

Comment:
Where is the rest of the sentence? I expected something like
```suggestion
Transparent pipelines for assessing online information quality
```
Taken from https://www.esciencecenter.nl/projects/the-eye-of-the-beholder-transparent-pipelines-for-assessing-online-information-quality/